src/assets/theme.js

```diff
@@ -44,4 +44,6 @@ const theme = buildTheme({
   animationSpeeds
 })
 
+console.info('THEME', theme)
```

Comment:
Please remove this or conditionally print it based on environment variable `NODE_ENV`: do not print anything in production builds.

Also, remember to run `npm run lint` prior to opening a PR

Reply:
Can we add husky for that?